scripts/pi-hole/php/add.php

```diff
@@ -44,12 +44,18 @@
             echo shell_exec("sudo pihole -a audit ".$domains);
         }
         break;
-    case "regex":
+    case "black_regex":
         echo shell_exec("sudo pihole --regex --web ".$domains);
         break;
-    case "wild":
+    case "white_regex":
```

Comment:
Can we add support for the whitelist regex to teleporter?

Reply:
Done, I'll mark this PR as `[WIP]` as this new feature might need more testing.